### ISSUE TITLE
Setting CustomTimeouts on AzureNextGen does not seem to have an effect

### ISSUE THREAD

**Issue by crAZiAc** [pulumi/pulumi-azure-nextgen#252]
When creating an Azure VMMS using AzureNextGen, setting the **CustomTimeouts** on the  **CustomResourceOptions** does not seem to have any effect.

```csharp
var vmScaleset = new AzureNextGen.Compute.Latest.VirtualMachineScaleSet
...

``` 
```csharp
new Pulumi.CustomResourceOptions
                    {
                           CustomTimeouts = new CustomTimeouts 
                        { 
                            Create = TimeSpan.FromHours(4),
                            Update = TimeSpan.FromHours(4)
                        }
                    }
```

Running an operation where I change the size (SKU) of the VMSS, results in this: 

```powershell
azure-nextgen:compute/latest:VirtualMachineScaleSet bd-sf-vmss-01-acc updating [diff: ~sku,tags]; error: Future#WaitForCompletion: context has been cancelled: StatusCode=200 -- Original Error: context deadline exceeded
azure-nextgen:compute/latest:VirtualMachineScaleSet bd-sf-vmss-01-acc **updating failed** [diff: ~sku,tags]; error: Future#WaitForCompletion: context has been cancelled: StatusCode=200 -- Original Error: context deadline exceeded

```

The operation to change the SKU on a VirtualMachineScaleSet running five ServiceFabric nodes usually takes little over 2 hours. It seems to timeout exactly at two hours, while I've added the CustomTimeOuts property to timeout at four hours.

Do the CustomTimeouts get sent down to the provider?

**Comment by mikhailshilkov** [comment]
I believe you are right and two hours are hard-coded in the provider, currently

**Comment by mikhailshilkov** [comment]
Will be fixed in 0.6.0
